src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/__tests__/utils/validation.test.ts

```diff
@@ -980,14 +980,45 @@ describe('>>> utils/validations.ts', () => {
     expect(validations).toEqual(expectedResult);
   });
 
-  it('+++ getNumberOfComponentsWithErrors should return correct number of components with error', () => {
-    const componentsWithErrors = validation.getNumberOfComponentsWithErrors(mockApiResponse);
-    expect(componentsWithErrors).toEqual(1);
-  });
 
-  it('+++ getNumberOfComponentsWithEWarnings should return correct number of components with warnings', () => {
-    const componentsWithWarnings = validation.getNumberOfComponentsWithWarnings(mockApiResponse);
-    expect(componentsWithWarnings).toEqual(1);
+  describe('hasValidationsOfSeverity', () => {
+    it('should return true when validations have errors and checking for Severity.Error', () => {
+      const result = validation.hasValidationsOfSeverity(mockValidations, Severity.Error);
+      expect(result).toBeTruthy();
+    });
+
+    it('should return false when validations have no errors and checking for Severity.Error', () => {
+      const validationsWithNoErrors: IValidations = {
+        page1: {
+          component1: {
+            simpleBinding: {
+              warnings: ['some warning'],
```

Comment:
This test would probably still pass if this warning was removed. 

I think it would be clearer if the test title was updated to something like `should return false when validations has warnings, and checking for Severity.Error`, and perhaps even add a separate test for `should return false when validations have no errors or warnings, and checking for Severity.Error`

The reason I think its important to split this up, and update the title is that currently the title seems to not care about warnings. If someone were to do some cleanup in this area later, this warning might get removed, since it seems redundant. Having 2 explicit tests, one about warnings and one about no errors or warnings makes it a bit clearer that both cases are important.